Ben wilson bylaws2.5 draft (#36)
cabforum/forum: Bylaws.md

Members of the CA/Browser Forum have worked closely together in defining the guidelines and means of implementation for best practices as a way of providing a heightened security for Internet transactions and creating a more intuitive method of displaying secure sites to Internet users.

## 1.2 Status of the Forum and Forum Activities

The Forum has no corporate or association status, but is simply a group of Certificate Issuers and Certificate Consumers that communicates or meets from time to time to discuss matters of common interest relevant to the Forum’s purposes. The Forum has no regulatory or industry powers over its members or others. Other than those rights and responsibilities found in the Forum’s Intellectual Property Rights (IPR) Policy, Forum “membership” or other participation status does not convey any legal status or rights, but is intended simply as a guide to the levels of participation in Forum activities.

## 1.3 Intellectual Property Rights Policy; Antitrust Laws and Regulations; Goal; Conduct

> Forum Voting Members, Non-Voting Members, Associate Members, Probationary Members, and Interested Parties must comply with the then-current IPR Policy, code of conduct, and all applicable antitrust laws and regulations during their Forum activities. Before an applicant for membership is evaluated to join a Chartered Working Group (CWG), it must be confirmed that the person signing the IPR agreement on behalf of an applicant is authorized to bind that entity to this agreement. 

Reviewer: ```suggestion
Forum Voting Members, Non-Voting Members, Associate Members, Probationary Members, and Interested Parties must comply with the then-current IPR Policy, Code of Conduct, and all applicable antitrust laws and regulations during their Forum activities. Before an applicant for membership is evaluated to join a Chartered Working Group (CWG), it must be confirmed that the person signing the IPR agreement on behalf of an applicant is authorized to bind that entity to this agreement. 
```

Categories of Forum Membership are defined as follows:

> * Voting Member, which contains the following two (2) sub-categories:
>    * Certificate Issuer: The member organization operates a certification authority and meets the requirements for membership as a Certificate Issuer in one or more CWGs.
>    * Certificate Consumer: The member organization produces software that processes certificates and meets the requirements for membership as a Certificate Consumer in one or more CWGs.
> Other types of membership consists of the following three (3) sub-categories:
>    * Associate Member: As defined in Section 3.1
>    * Interested Party: As defined in Section 3.2
>    * Probationary Member (Certificate Issuer/Certificate Consumer): As defined in Section 3.3

Reviewer: The highlighted text should be applicable to both types of voting members (at least for the probationary member) and not just Certificate Consumers, but its written (but not listed) as if this is a subcategory of a "voting member"?

I have included a suggestion below.

![image](https://github.com/cabforum/forum/assets/722070/2c4b3000-85c1-48e9-8a96-b2070e827a3f)

```suggestion
* Voting Member, which contains the following two (2) sub-categories:
   * Certificate Issuer: The member organization operates a certification authority and meets the requirements for membership as a Certificate Issuer in one or more CWGs.
   * Certificate Consumer: The member organization produces software that processes certificates and meets the requirements for membership as a Certificate Consumer in one or more CWGs.
* Probationary Member: As defined in Section 3.3
   * Certificate Issuer: The member organization operates a certification authority and does not meet all qualifying criteria for membership as a Certificate Issuer in one or more CWGs.
   * Certificate Consumer: The member organization produces software that processes certificates and does not meet all qualifying criteria for membership as a Certificate Consumer in one or more CWGs.
* Associate Member: As defined in Section 3.1
* Interested Party: As defined in Section 3.2
```

Categories of Forum Membership are defined as follows:

* Voting Member, which contains the following two (2) sub-categories:
   * Certificate Issuer: The member organization operates a certification authority and meets the requirements for membership as a Certificate Issuer in one or more CWGs.
   * Certificate Consumer: The member organization produces software that processes certificates and meets the requirements for membership as a Certificate Consumer in one or more CWGs.

Non-Voting Membership consists of the following four (4) sub-categories:
   * Associate Member: As defined in Section 3.1
   * Interested Party: As defined in Section 3.2
   * Probationary Member: As defined in Section 3.3
   * Non-Voting Member: As defined in Section 3.4
 
Note: Based on section 3.1 in version 2.4 of the Bylaws, it was allowed for a Certificate Issuer that partially met the CWG membership criteria to be accepted as "Associate Member". Once version 2.5 of the Bylaws becomes effective, these Associate Members will automatically be re-assigned to the "Probationary Member" membership category, as desctibed in Section 3.3 below.

> Voting Members that qualify under more than one category of Forum Membership MAY choose only one membership category at the Forum level.

Reviewer: ```suggestion
Voting Members that qualify under more than one category of Forum Membership MAY choose one membership category at the Forum level.
```

 8. If no Exclusion Notices are filed during the Review Period with respect to a Draft Guideline Ballot, then the results of the Initial Vote are automatically deemed to be final, approved, and effective as of the end of the applicable Review Period. The Draft Guidelines then become either Final Guidelines or Final Maintenance Guidelines, as designated in the Draft Guidelines Ballot. The Chair will notify the Public Mail List of the results of the Review Period within three (3) business days, as well as update the Public Website of Final Guidelines and Final Maintenance Guidelines within ten (10) business days, of the close of the Review Period. The Chair or Vice-Chair of the Forum or of the CWG is allowed to perform changes to informative (non-normative) parts of a Final Guideline or Final Maintenance Guideline before it is published to the public web site and without requiring a ballot procedure. The set of changes are limited to:
    * The cover page,
    * The Table of Contents,
    * The year in the “Copyright” information,

>     * Footers with page numbers

Reviewer: We know have many internal links that make it easier to navigate to other sections, I think it must be possible to include/remove/update these internal links without a ballot, as this is just document formatting.
```suggestion
    * Footers with page numbers
    * Links to other sections within the document
```

Reviewer 2: Sounds reasonable. Added in https://github.com/cabforum/forum/commit/c6466604aa38e7c63ea4833aeea432081fb4bdf8

    
    The Chair or Vice-Chair of the Forum or of a CWG is also allowed to perform the following changes, unless the ballot explicitly updates this information:
   
    * The Guideline version number,

>     * Headers/Footers with version numbers,

Reviewer: Same as above.

```suggestion
    * Headers/Footers with version numbers,
    * Links to other sections within the document,
```

## 3.1 Associate Members

> Membership in the Forum as an Associate Member may be by invitation when the CA/Browser Forum determines that maintaining such a relationship will be of benefit to the work of the Forum. In the past, entities qualifying as Associate Members have included the ACAB'c, AICPA/CICA WebTrust Task Force, the European Telecommunications Standards Institute (ETSI), the Internet Corporation for Assigned Names and Numbers, tScheme and the U.S. Federal PKI. In order to become an Associate Member, an organization must sign a mutual letter of intent/understanding, or other similar agreement, and the Forum’s IPR Policy Agreement, unless this latter requirement is waived in writing by the Forum based on overriding policies of the Associate Member’s own organization IPR rules. 

Reviewer: ```suggestion
Membership in the Forum as an Associate Member may be by invitation when the CA/Browser Forum determines that maintaining such a relationship will be of benefit to the work of the Forum. In the past, entities qualifying as Associate Members have included the ACAB'c, AICPA/CICA WebTrust Task Force, the European Telecommunications Standards Institute (ETSI), the Internet Corporation for Assigned Names and Numbers, tScheme and the U.S. Federal PKI. In order to become an Associate Member, an organization must sign a mutual letter of intent/understanding, or similar agreement, and the Forum’s IPR Policy Agreement, unless this latter requirement is waived in writing by the Forum based on overriding policies of the Associate Member’s own organization IPR rules. 
```

[The following is a proposed text for new Chartered Working Groups that can be amend accordingly].

> (a) The Working Group shall consist of two classes of voting members--Certificate Issuers and Certificate Consumers meeting the following criteria:

Reviewer: What is the intention of '--', should we just replace with this a line ending?

```suggestion
(a) The Working Group shall consist of two classes of voting members. Certificate Issuers and Certificate Consumers meeting the following criteria:
```

7. **Decision Process**

> This Working Group will seek to make decisions when there is consensus and with due process. The expectation is that, typically, the Chair or other participant makes an initial proposal, which is then refined in discussion with the Working Group participants, and consensus emerges with little formal voting being required. However, if a decision is necessary for timely progress, but consensus is not achieved after careful consideration of the range of views presented, the Chair(s) should put the question out for voting within the WG (using email and/or web-based survey techniques) according to Section 2 (Forum Membership and Voting) of the Forum Bylaws and record a decision, along with any objections. The matter should then be considered resolved unless and until new information becomes available.

Reviewer: Was it 'Chair(s)' to include the Chair as well as the Vice-chair?
